.github/workflows/lint-and-test.yml

```diff
@@ -104,6 +104,19 @@ jobs:
         run: cargo test -p proof-of-sql --no-run --no-default-features --features="std"
       - name: Run cargo test (proof primitives - Dory) (std feature only - i.e. not using blitzar)
         run: cargo test proof_primitive::dory::dory_compute_commitments_test --no-default-features --features="std"
+      - uses: taiki-e/install-action@cargo-llvm-cov
+      - name: Run Cargo Test with Coverage
+        run: cargo llvm-cov --workspace --lcov --output-path lcov.info
```

Comment:
I would like to see this as a separate job, and not part of the Test Suite since we will likely want to put it in evaluation mode for a while before we make it a requirement for merging a PR.